### DISCUSSION TITLE
kafka-ui api + Prometheus + Grafana

### DISCUSSION THREAD

**Discussion by rja1** [provectus/kafka-ui#2853] [Q&A] [answered]
Curious about using kafka-ui api + Prometheus + Grafana for consumer-lag trending / monitoring.  In doing some searching, looks like this was discussed a while back (https://github.com/provectus/kafka-ui/issues/942 & https://github.com/provectus/kafka-ui/issues/86).  Was this ever actually implemented?  If so, does anyone have any examples?

Thanks,
Rob

**Comment by Haarolean** [comment, marked as answer]
Hey, #942 is still open, it's on our roadmap.
